src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+- Make passwords as long as allowed by either your password generator or the form you're filling out.
+- Use a mixture of uppercase, lowercase, numbers and symbols
+- Avoid common dictionary words
+
+[More on creating strong passwords](https://terranovasecurity.com/how-to-create-a-strong-password-in-7-easy-steps/)
+
+### Use unique passwords for everything
+
+A strong password doesn't provide as much protection if the password is revealed in a data breach. The website [Have I Been Pwned](https://haveibeenpwned.com) allows you to check if your accounts were involved in any data breaches stored in their database. If they have, **you should change pwned passwords immediately**. Using unique passwords for every account lowers the risk of hackers getting access to all of your accounts when one of your passwords is compromised.
+
+### Use a password manager
+
+<InfoBanner emoji=":bulb:">
+  <div>
+    Using a password manager takes care of creating strong, unique passwords and remembering them! We <strong>strongly</strong> recommend using one.
+  </div>
+</InfoBanner>
+
+Remembering strong, unique passwords for every account you have isn't ideal. A password manager offers a secure, encrypted store for all your passwords that you can access through one strong master password. They also suggest strong passwords when signing up for a new service, so you don't have to create your own. Many password managers will also tell you if you have been involved in a data breach, allowing you to change the passwords before any malicious attacks.
+
+![Example of using a password manager](./passwordManager.png)
+
+#### Try a password manager:
+
+- [Bitwarden](https://bitwarden.com/)
+- [KeePass](https://keepass.info/)
+- [LastPass](https://www.lastpass.com/)
+- [1Password](https://1password.com/)
+
+### Use Two-Factor Authentication
+
+To prove you are you there are different unique proofs that can be used for authentication. These are known as factors and the three main factors are:
+
+- something you know (your password)
+- something you are (such as a fingerprint or iris scanner)
+- and something you own (a security key)
+
+Using **Two-Factor Authentication (2FA)** provides an additional *security factor* for your online accounts so that knowing your password alone (something you know) is not enough to access an account. Most commonly, the second factor is a randomised 6-digit code (something you own) that you can access through an authenticator app such as Google Authenticator or Authy. These work as a "something you own" factor because the seed is stored on your device.
+
+<InfoBanner emoji=":lock:">
+  <div>
+    Note: Using SMS-based 2FA is susceptible to 
+    <a href="https://www.vice.com/en/article/3kx4ej/sim-jacking-mobile-phone-fraud">
+      SIM jacking
+    </a>
+     and is not secure. For the best security, use a service like{" "}
+    <a href="https://mashable.com/article/how-to-set-up-google-authenticator">
+      Google Authenticator
```

Comment:
While authenticator apps are the most popular and many people likely consider a security key too expensive, it would probably still make sense to mention them and note that this is the safest 2FA option.